### PR TITLE
AttributeFilter styles alignments

### PR DESCRIPTION
**Pull request by kandl**
- align styles with UX designs
- make a bit order in CSS classes
- complete mobile styles (with respect to fullscreenOnMobile prop)
- fix condition for select all checkbox deselecting items in InvertableSelect
- fix attribute elements loading on tiger/panther

JIRA: RAIL-4408

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx [hunk 1]
```diff
@@ -27,41 +29,52 @@ export const AttributeFilterDropdown: React.VFC = () => {
...
         onApply,
+        fullscreenOnMobile,
     } = useAttributeFilterContext();
 
+    const isMobile = useMediaQuery("mobileDevice");
+
     return (
         <Dropdown
-            className="gd-attribute-filter-dropdown__next"
+            className={cx(["gd-attribute-filter__next"])}
```
Comment (ivanmjartan) [inline libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx:39]: remove this cx function

File: libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx [hunk 2]
```diff
@@ -27,41 +29,52 @@ export const AttributeFilterDropdown: React.VFC = () => {
...
-                    title={title}
-                    subtitle={subtitle}
-                    isFiltering={isFiltering}
-                    isLoaded={!isInitializing && !initError}
-                    isLoading={isInitializing}
-                    isOpen={isOpen}
-                    selectedItemsCount={committedSelectionElements.length}
-                    onClick={toggleDropdown}
-                />
+                <div className={cx({ "gd-is-mobile": fullscreenOnMobile && isMobile && isOpen })}>
```
Comment (ivanmjartan) [inline libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx:47]: put this style calculation into const

File: libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx [hunk 3]
```diff
@@ -27,41 +29,52 @@ export const AttributeFilterDropdown: React.VFC = () => {
...
             renderBody={({ closeDropdown }) => (
-                <DropdownBodyComponent
-                    onApplyButtonClick={() => {
-                        onApply();
-                        closeDropdown();
-                    }}
-                    onCloseButtonClick={closeDropdown}
-                />
+                <div
+                    className={cx({ "gd-is-mobile": fullscreenOnMobile && isMobile })}
```
Comment (ivanmjartan) [inline libs/sdk-ui-filters/src/AttributeFilter@next/Components/Dropdown/AttributeFilterDropdown.tsx:67]: maybe this cx stuff could be in const

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9397/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11240/ : FAILURE in 41s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9399/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11242/ : FAILURE in 19m 44s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9406/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11251/ : FAILURE in 21m 33s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/9410/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11256/ : SUCCESS in 20m 01s

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2879

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2656/ : SUCCESS in 2m 15s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2957/ : SUCCESS in 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1790/ : SUCCESS in 18m 40s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11260/ : FAILURE in 28m 05s

**Review by ivanmjartan — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2879

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2657/ : SUCCESS in 2m 22s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2958/ : SUCCESS in 21s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1791/ : SUCCESS in 14m 56s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/11262/ : SUCCESS in 20m 02s

**Pull request merged**